lib/routes/swpu/bgw.ts

```diff
@@ -29,21 +29,21 @@ export const route: Route = {
     maintainers: ['CYTMWIA'],
     handler,
     url: 'swpu.edu.cn/',
-    description: `| 栏目 | 重要通知公告 | 部门通知公告 | 本周活动 | 学术报告 |
-  | ---- | ------------ | ------------ | -------- | -------- |
-  | 代码 | zytzgg       | bmtzgg       | bzhd     | xsbg     |`,
+    description: `| 栏目 | 重要通知公告 | 部门通知公告 | 本周活动 |
+  | ---- | ------------ | ------------ | -------- |
+  | 代码 | zytzgg       | bmtzgg       | bzhd     |`,
 };
 
-async function handler(ctx) {
-    const url = `https://www.swpu.edu.cn/bgw2/${ctx.req.param('code')}.htm`;
+async function handler(ctx): Promise<Data> {
+    const url = `https://www.swpu.edu.cn/bgw/${ctx.req.param('code')}.htm`;
 
     const res = await got.get(url);
     const $ = load(res.data);
 
     const title = $('.title').text();
 
     // 获取标题、时间及链接
-    const items = [];
+    const items: DataItem[] = [];
     $('.notice > ul > li > a').each((i, elem) => {
         items.push({
             title: $(elem.children[0]).text(),
```

Comment:
Please update it to use [map()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) as well.